elasticManager.js

```diff
@@ -42,16 +42,31 @@ class ElasticManager {
     delete changeStreamObj.fullDocument._id;
     const esReadyDoc = changeStreamObj.fullDocument;
 
+    let parentId = this.getParentId(esReadyDoc, this.mappings[changeStreamObj.ns.coll].type);
+
     this.bulkOp.push({
         index:  {
           _index: this.mappings[changeStreamObj.ns.coll].index,
           _type: this.mappings[changeStreamObj.ns.coll].type,
-          _id: esId
+          _id: esId,
+          _parent:parentId
```

Comment:
parentId being null doesn't cause any errors or unintended behavior right?

Reply:
Nope.  In fact it doesn't even send across the network to ES.